Mobile Test Runner for Jama/Mobile Test Runner for Jama/Base.lproj/Main.storyboard

```diff
@@ -956,7 +963,7 @@
                         </constraints>
                     </view>
                     <toolbarItems/>
-                    <navigationItem key="navigationItem" title="Test Run Index" id="Bjc-ta-NLl">
+                    <navigationItem key="navigationItem" title="Test Run" id="Bjc-ta-NLl">
```

Comment:
I changed the title on the index page to match the back button on the step detail page. It also makes it less crowded on smaller screens. Not 100% if this is what we want to do though. Any input would be great.